lib/libutee/include/user_ta_header.h

```diff
@@ -46,6 +43,12 @@
 	 */
 #define TA_FLAG_CONCURRENT		(1 << 8)
 
+#define TA_FLAGS_MASK		GENMASK_32(8, 2)
+
+/* Deprecated IDs that previously were meaningless */
+#define TA_FLAG_USER_MODE		0
```

Comment:
Why do we keep these?
When are we going to remove them of not when increasing major number?

Reply:
This is mainly about syncing the several TA sources.
I though leaving these definition for a while before we fully remove them, for users to not have a surprise with TA source that now refuse to compile because we removed a macro. For example: 3.0.0, deprecate the use. 3.1.0 fully remove the definition.

Reply:
Fair enough.
Please add a comment when these are supposed to be removed.

Reply:
`These macros will be definitely removed in a near future.`
Is it ok? or do you expect something like `during the 3.x.x revisions.`.

Reply:
Yes, if it's next release then write something like:
`/* Deprecated flags that will be removed in the 3.1 release */`